shared/ui/votingLink/votingLink.tsx

```diff
@@ -0,0 +1,27 @@
+import { Link } from '@lidofinance/lido-ui';
+import { FC, PropsWithChildren, memo, useEffect, useState } from 'react';
+
+export const VotingLink: FC<PropsWithChildren<{ voteId?: string }>> = memo(
+  ({ voteId, children }) => {
+    const [href, setHref] = useState('https://vote.lido.fi');
+
+    useEffect(() => {
+      const url = new URL(window.location.href);
+      const tld =
+        url.hostname === 'localhost'
+          ? 'lido.fi'
+          : url.hostname.split('.').slice(-2).join('.');
```

Comment:
will this work OK with preview stands?

Reply:
We don't have preview stands on TRP now and probably there will be changes of how do they work in some foreseeable future 🤔 So I'm not sure if we need to add this case ahead of time

Reply:
Do you think it's reasonable only allow these changes for prod/testnet/stage and keep vote.lido.fi for all other runs?